config/environments/production.rb

```diff
@@ -21,10 +25,10 @@
 
   # Disable serving static files from the `/public` folder by default since
   # Apache or NGINX already handles this.
-  config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
+  # config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
```

Comment:
do you still want to disable these?

Reply:
It was needed to serve the assets in production